translations/es/README.md

```diff
@@ -0,0 +1,383 @@
+
+El repositorio contiene lo siguiente:
+
+* `/.devcontainer`
+   - `.devcontainer/Dockerfile`: archivo de configuración utilizado por Codespaces para determinar el sistema operativo y otros detalles.
+   - `.devcontainer/devcontainer.json`: archivo de configuración utilizado por Codespaces para configurar la configuración del código de Visual Studio, como la habilitación de extensiones adicionales.
+   - `.devcontainer/post-create.sh`: archivo de configuración utilizado por Codespaces para instalar herramientas adicionales, como PowerShell.
+* `/src`: Proyecto Blazor WebAssembly para construir su sitio de portafolio.
+* `.editorconfig`: Configuración para [EditorConfig](https://editorconfig.org/) que ayuda a mantener estilos de codificación consistentes en Codespaces.
+* `global.json`: Configuración para la aplicación Blazor WebAssembly para evitar el uso de la versión .NET pre-lanzada.
+* `swa-cli.config.json`: Configuración para [Azure SWA CLI](https://azure.github.io/static-web-apps-cli/) para ejecutar la aplicación Blazor WebAssembly en Codespaces.
+* `MyPortfolio.sln`: el archivo de solución que contiene el proyecto de aplicación Blazor WebAssembly.
+
+<br />
+
+## 🚀 Comenzando
+
+Este proyecto de sitio de portafolio está lleno de datos de muestra para que pueda abrir Codespaces inmediatamente, verlo en ejecución e implementarlo en cualquier momento.
+
+Su entorno de desarrollo está listo para que comience. Basado en nuestra [plantilla de .NET CodesPaces](https://github.com/education/codespaces-teaching-template-dotnet), aquí está lo que ya está configurado y listo para que use:
+
+* Simple [Blazor WebAssembly](https://dotnet.microsoft.com/apps/aspnet/web-apps/blazor?wt.mc_id=dotnet-82024-juyoo) Aplicación con componentes para cada sección del sitio de portafolio
+* [SWA CLI](https://azure.github.io/static-web-apps-cli/) en su lugar para construir su sitio cuando se implementa
+* Código linting y en formato usando [EditorConfig](https://editorconfig.org/) para la consistencia del código.
+
+### Crea tu portafolio
+
+1. Cree un repositorio a partir de esta plantilla. Use este [crear enlace de repositorio](https://github.com/education/codespaces-teaching-template-dotnet/generate). Seleccione el propietario del repositorio, proporcione un nombre, una descripción si lo desea y si desea que el nuevo repositorio sea público o privado.
+2. Navegue a la página principal del repositorio recién creado.
+3. Debajo del nombre del repositorio, use el menú desplegable Código y, en la pestaña Espacios de códigos, seleccione "Crear codespace en principal".
+
+     <img src="https://docs.github.com/assets/cb-138303/images/help/codespaces/new-codespace-button.png" alt="Crear codespace" style="width:270px;"/>
```

Comment:
this image doesn't show up, we need some help, please @cyz🥺

Reply:
@LucioMSP please, can you use this image: https://user-images.githubusercontent.com/2198735/250924075-7e35b981-b983-47d4-af9e-f2e0af207103.png

Reply:
Ok... done